tests/test_relay.py

```diff
@@ -0,0 +1,57 @@
+import strawberry
+from strawberry import auto, relay
+from strawberry.schema.config import StrawberryConfig
+
+import strawberry_django
+
+from .models import User
+
+
+def test_relay_with_nodeid():
+    @strawberry_django.type(User)
+    class UserType(relay.Node):
+        id: relay.NodeID[int]
+        name: auto
+
+    @strawberry.type
+    class Query:
+        user: UserType
+
+    schema = strawberry.Schema(query=Query, config=StrawberryConfig(False))
+    str(schema)
```

Comment:
Left over

Reply:
nope ;).
It is for testing/executing print_schema.

Reply:
Oh, I see. So probably you want to compare it to the expected value. If the schema was generated it will print, the main thing to test here is that it prints what we expect

Check `tests/filters/test_filters.py` on function `test_pk_inserted_for_root_field_only` for an example